### PR TITLE
Update languageName = हिनदी in config.toml to 'हिनदी(Hindi)'

### LINKED ISSUE #35783: [hi] Update `languageName = "हिन्दी"` in config.toml to implement recent changes in the language name standard.
**This is a Bug Report**

**Problem:**

`config.toml` in `dev-1.22-hi.1` contains `languageName = "हिन्दी"`
`config.toml` in `main` contains  `languageName = "हिन्दी (Hindi)"`, which conforms to recent changes in the recent [language name standard](https://github.com/kubernetes/website/blob/main/content/en/docs/contribute/localization.md#modify-the-site-configuration). Also see #31657

**Proposed Solution:**

- We should update this line in `dev-1.22-hi.1` so it is the same as `main`.

**Page to Update:**

`config.toml`

### PR DESCRIPTION
**Pull request by xzyangNB**
Fix #35783

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
LGTM

File: config.toml
Comment (Babapool) [inline config.toml:445]: Just a small nit, but otherwise LGTM:
```suggestion
languageName = "हिन्दी (Hindi)"  
```

**Comment by xzyangNB** [comment]
I don't know why this PR has not been merged. What is the situation now?

**Comment by sftim** [comment]
> I don't know why this PR has not been merged. What is the situation now?

The Hindi localization team must review this PR. Once that has happened, we can merge it.

**Comment by Babapool** [comment]
The changes LGTM.  (Unofficial LGTM from my side.. I am not a `hi` localization approver). But I am happy to ask the reviewers to have a look at this PR, if they don't visit it a few days. Thank you @xzyangNB for your contribution and patience.

**Comment by xzyangNB** [comment]
@tengqm  Hello, this PR has been for a while. Just now GitHub sent me an e-mail prompt to assign it to you. Please help me deal with it. Thank you

**Review by anubha-v-ardhan — APPROVED** [review]
/assign
/lgtm
/approve

Thank you @xzyangNB :)

**Comment by tengqm** [comment]
/lgtm
/approve

I don't speak Hindi, but I'm sure this PR is correct. :)

**Pull request merged**
